src/Mod/Draft/draftguitools/gui_snap.py

```diff
@@ -0,0 +1,281 @@
+# ***************************************************************************
+# *   (c) 2009 Yorik van Havre <yorik@uncreated.net>                        *
+# *   (c) 2010 Ken Cline <cline@frii.com>                                   *
+# *   (c) 2019 Eliud Cabrera Castillo <e.cabrera-castillo@tum.de>           *
+# *   (c) 2020 Carlo Pavan                                                  *
+# *                                                                         *
+# *   This file is part of the FreeCAD CAx development system.              *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   FreeCAD is distributed in the hope that it will be useful,            *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with FreeCAD; if not, write to the Free Software        *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+"""This module provides the Draft Snap Gui commands.
+"""
+## @package gui_base
+# \ingroup DRAFT
+# \brief This module provides the Draft Snap Gui commands.
+
+import FreeCAD as App
+import FreeCADGui as Gui
+import DraftGui
```

Comment:
We don't want to import `DraftGui` if we can avoid it. See #3091. Importing `DraftGui` sets up the toolbar, so it has to be imported only once, in `DraftTools.py`.